lib/kontrast/configuration.rb

```diff
@@ -71,11 +71,12 @@ def validate
             end
         end
 
-        def pages(width)
+        def pages(width, url_params = {})
```

Comment:
I'm okay with dropping 1.9 support. I would also prefer default value (at least here) since it makes the actual usage of it much easier:

```
config.pages(320, mobile: 1, "optimizely_disable": true, "20150701_product_reviews": "no_reviews") do |page|
    page.home "/"
end
```

vs.

```
config.pages(320, url_params: { mobile: 1, "optimizely_disable": true, "20150701_product_reviews": "no_reviews" }) do |page|
    page.home "/"
end
```